src/plotting.jl

```diff
@@ -0,0 +1,7 @@
+# Note: type-recipes seem to need to use short-form functions
+@recipe function f(::Type{<:ZonedDateTime}, val::ZonedDateTime)
+    return (
+        zdt -> Dates.value(DateTime(zdt, UTC)),
+        dt -> string(DateTime(Dates.UTM(dt))),
+    )
+end
```

Comment:
Can you go into how this works? I would like to have the time zone appear in some form. Probably just the numeric offset would be a good place to start.

Reply:
I have no idea how this one works internally.
but the effect is to convert it ot UTC.
And in that case I think showing the timezone is not required. UTC the sensible default.
(One could even argue that all `DateTimes` are in UTC, since with have `datetime2unixepoch` and `unixepoch` is defined as being in UTC, right?)


If you want to show the timezone,
then I think we should not convert to UTC, but should convert to the timezone of the first element, then use `Local` time, and display the timezone in the axis label.
But that would be a totally differnent PR.

Reply:
I think end users will just assume either UTC or Local depending on their personal preference. As I said I'd like to show the time zone offset to avoid any confusion that could occur.

> (One could even argue that all DateTimes are in UTC, since with have datetime2unixepoch and unixepoch is defined as being in UTC, right?)

`DateTime`s have an implicit time zone but without context you don't know what that is. `2020-03-16T12:00` to me means noon in *any* time zone until additional context is provided.

Reply:
Shall I close this PR then and make another one?

Reply:
Maybe make the other PR first. I'm unclear on why this PR shouldn't be just updated

Reply:
It will use a entirely different mechanism. 
A user-recipe rather than a type-recipe, I think is the terminolgy.
Not even sure the tests will be the same.
But I can do it in this one.